### PR TITLE
Don't have `testkomodo.sh` automatically run code

### PR DESCRIPTION
**Pull request by pinkwah**
This breaks all current `testkomodo.sh` scripts, but it should make them much simpler in the long run. `testkomodo.sh` scripts can expect `ci_helper.sh` functions to be available with the prefix `ci_`, but other than that it is up to each and every script to handle the setup and testing. `run_tests` is no longer a function that is ran.

### CODE REVIEW

File: vars/testKomodo.groovy [hunk 1]
```diff
@@ -137,7 +137,6 @@ def call(Map args = [:]) {
                         source ${env.CI_SOURCE_ROOT}/${params.TEST_SCRIPT}
```
Comment (sondreso) [inline vars/testKomodo.groovy:137]: IS the plan that the test script will run whatever it wants to run when sourced?

**Review by sondreso — COMMENTED** [review]
I think the changes look OK, but when merged, will this not break testing of every current release? :thinking:

**Comment by jonathan-eq** [comment]
Is this still being worked on, or can it be closed? @pinkwah

**Comment by pinkwah** [comment]
Stale.

**Pull request closed without merging**